--- setup.py	2014-02-04 15:06:55.000000000 -0800
+++ setup.py	2014-02-04 15:07:14.000000000 -0800
@@ -112,7 +112,7 @@
         # #additiona-meta-data note #3
         "url": 'http://ged.msu.edu/',
         "packages": ['khmer'],
-        "install_requires": ['screed >= 0.7', 'argparse >= 1.2.1', ],
+        "install_requires": ['screed >= 0.7'],
         "setup_requires": ['nose >= 1.0', 'sphinx', ],
         "scripts": scripts,
         "ext_modules": [extension_mod, ],
