--- core/mafft.tmpl	2013-12-30 18:56:04.000000000 -0800
+++ core/mafft.tmpl	2013-12-30 18:57:47.000000000 -0800
@@ -1,4 +1,6 @@
-#! /bin/sh
+#! /bin/bash
+
+bindir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"; export bindir
 
 er=0;
 myself=`dirname "$0"`/`basename "$0"`; export myself
@@ -24,7 +26,7 @@
 if [ "$MAFFT_BINARIES" ]; then
 	prefix="$MAFFT_BINARIES"
 else
-	prefix=_LIBDIR
+	prefix=$bindir/../libexec/mafft
 fi
 export prefix
 
