--- core/Makefile	2014-04-26 16:38:17.000000000 -0700
+++ core/Makefile	2014-04-26 17:18:06.000000000 -0700
@@ -1,11 +1,11 @@
-PREFIX = /usr/local
+PREFIX = $(PRFX)
 LIBDIR = $(PREFIX)/libexec/mafft
 BINDIR = $(PREFIX)/bin
 MANDIR = $(PREFIX)/share/man/man1
 
 #MNO_CYGWIN = -mno-cygwin
 
-ENABLE_MULTITHREAD = -Denablemultithread
+# ENABLE_MULTITHREAD = -Denablemultithread
 # Comment out the above line if your compiler 
 # does not support TLS (thread-local strage).
 
@@ -133,7 +133,7 @@
 	sed "s:_PROGS:$(PROGS):" univscript.tmpl  > univscript
 
 mafft: mafft.tmpl mltaln.h
-	sed "s:_LIBDIR:$(LIBDIR):" mafft.tmpl  > mafft
+	cat mafft.tmpl > mafft
 
 mafft-homologs.rb: mafft-homologs.tmpl
 #	cp mafft-homologs.tmpl mafft-homologs.rb
@@ -521,8 +521,8 @@
 ln -s mafft mafft-linsi; ln -s mafft mafft-ginsi; ln -s mafft mafft-fftns; \
 ln -s mafft mafft-fftnsi; ln -s mafft mafft-nwns; ln -s mafft mafft-nwnsi; \
 ln -s mafft mafft-einsi; ln -s mafft mafft-xinsi; ln -s mafft mafft-qinsi;\
-rm -f mafft-profile mafft-profile.exe; ln -s $(LIBDIR)/mafft-profile .; \
-rm -f mafft-distance mafft-distance.exe; ln -s $(LIBDIR)/mafft-distance . )
+rm -f mafft-profile mafft-profile.exe; ln -s ../libexec/mafft-profile .; \
+rm -f mafft-distance mafft-distance.exe; ln -s ../libexec/mafft-distance . )
 
 	mkdir -p $(MANDIR)
 	chmod 755 $(MANDIR)
