--- setup.py	2014-07-16 09:48:12.000000000 -0700
+++ setup.py	2014-07-16 10:05:13.000000000 -0700
@@ -26,40 +26,19 @@
 ###############################################################################
 # setuptools/distutils/etc. import and configuration
 
-try:
-    import ez_setup
-    try:
-        ez_setup_path = " ('" + os.path.abspath(ez_setup.__file__) + "')"
-    except OSError:
-        ez_setup_path = ""
-    sys.stderr.write("using ez_setup%s\n" %  ez_setup_path)
-    ez_setup.use_setuptools()
-    import setuptools
-    try:
-        setuptools_path = " ('" +  os.path.abspath(setuptools.__file__) + "')"
-    except OSError:
-        setuptools_path = ""
-    sys.stderr.write("using setuptools%s\n" % setuptools_path)
-    from setuptools import setup, find_packages
-except ImportError, e:
-    sys.stderr.write("using distutils\n")
-    from distutils.core import setup
-    sys.stderr.write("using canned package list\n")
-    PACKAGES = ['dendropy',
-                'dendropy.dataio',
-                'dendropy.dataobject',
-                'dendropy.test',
-                'dendropy.test.support',
-                'dendropy.utility']
-    EXTRA_KWARGS = {}
-else:
-    sys.stderr.write("searching for packages\n")
-    PACKAGES = find_packages()
-    EXTRA_KWARGS = dict(
-        install_requires = ['setuptools'],
-        include_package_data=True,
-        test_suite = "dendropy.test"
-    )
+
+sys.stderr.write("using distutils\n")
+from distutils.core import setup
+sys.stderr.write("using canned package list\n")
+PACKAGES = ['dendropy',
+            'dendropy.dataio',
+            'dendropy.dataobject',
+            'dendropy.interop',
+            'dendropy.mathlib',
+            'dendropy.test',
+            'dendropy.test.support',
+            'dendropy.utility']
+EXTRA_KWARGS = {}
 
 PACKAGE_DIRS = [p.replace(".", os.path.sep) for p in PACKAGES]
 PACKAGE_INFO = [("% 40s : %s" % p) for p in zip(PACKAGES, PACKAGE_DIRS)]
@@ -82,23 +61,18 @@
 ###############################################################################
 # setuptools/distuils command extensions
 
-try:
-    from setuptools import Command
-except ImportError:
-    sys.stderr.write("setuptools.Command could not be imported: setuptools extensions not available\n")
+sys.stderr.write("setuptools command extensions are available\n")
+command_hook = "distutils.commands"
+ENTRY_POINTS[command_hook] = []
+
+###########################################################################
+# coverage
+from dendropy.test.support import coverage_analysis
+if coverage_analysis.DENDROPY_COVERAGE_ANALYSIS_AVAILABLE:
+    sys.stderr.write("coverage analysis available ('python setup.py coverage')\n")
+    ENTRY_POINTS[command_hook].append("coverage = dendropy.test.support.coverage_analysis:CoverageAnalysis")
 else:
-    sys.stderr.write("setuptools command extensions are available\n")
-    command_hook = "distutils.commands"
-    ENTRY_POINTS[command_hook] = []
-
-    ###########################################################################
-    # coverage
-    from dendropy.test.support import coverage_analysis
-    if coverage_analysis.DENDROPY_COVERAGE_ANALYSIS_AVAILABLE:
-        sys.stderr.write("coverage analysis available ('python setup.py coverage')\n")
-        ENTRY_POINTS[command_hook].append("coverage = dendropy.test.support.coverage_analysis:CoverageAnalysis")
-    else:
-        sys.stderr.write("coverage analysis not available\n")
+    sys.stderr.write("coverage analysis not available\n")
 
 
 ###############################################################################
